apiserver/stateauthenticator/authenticator_test.go

```diff
@@ -133,7 +135,12 @@ func (s *agentAuthenticatorSuite) setupMocks(c *gc.C) *gomock.Controller {
 
 	s.userService = NewMockUserService(ctrl)
 
-	authenticator, err := NewAuthenticator(s.StatePool, s.State, s.controllerConfigService, s.userService, s.agentAuthenticatorFactory, clock.WallClock)
+	s.bakeryConfigService = NewMockBakeryConfigService(ctrl)
+	s.bakeryConfigService.EXPECT().GetLocalUsersKey(gomock.Any()).Return(bakery.MustGenerateKey(), nil).AnyTimes()
```

Comment:
`AnyTimes` is any number including zero. I'd prefer to be a _little_ more specific.

Reply:
Amended.

`GetExternalUsersThirdPartyKey` has been kept `AnyTimes()` though, because it is not needed in every test